charts/datadog/templates/kube-state-metrics-core-rbac.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.datadog.kubeStateMetricsCore.enabled }}
+{{- if and (eq .Values.datadog.kubeStateMetricsCore.enabled true) (eq .Values.datadog.kubeStateMetricsCore.rbac.create true) }}
```

Comment:
why not?
{{- if and .Values.datadog.kubeStateMetricsCore.enabled .Values.datadog.kubeStateMetricsCore.rbac.create }}

Reply:
yes you’re right. I am going to make the change

Reply:
done @clamoriniere